flocker/node/agents/cinder.py

```diff
@@ -356,17 +373,14 @@ def create_volume(self, dataset_id, size):
             CLUSTER_ID_LABEL: unicode(self.cluster_id),
             DATASET_ID_LABEL: unicode(dataset_id),
         }
-        action_type = u"blockdevice:cinder:create_volume"
-        with start_action(action_type=action_type):
-            requested_volume = self.cinder_volume_manager.create(
-                size=int(Byte(size).to_GiB().value),
-                metadata=metadata,
-            )
-            Message.new(blockdevice_id=requested_volume.id).write()
```

Comment:
Removing the `start_action` makes sense because there is now automatic logging applied by the logging decorator.  But the logging decorator doesn't log the resulting `blockdevice_id`, I think.  It would be nice to keep that in the logs to be able to easily see where various Cinder volumes came from.

Simply leaving the `Message...` line in may be good enough for this.  It will get logged in the context of some wider Eliot action, I think.

Unless you think that information is available somewhere else already?